packages/sync-repo-settings/src/sync-repo-settings.ts

```diff
@@ -113,10 +113,11 @@ export function handler(app: Application) {
       }
 
       if (languageConfig[language]?.repoOverrides) {
-        const customConfig = languageConfig[language].repoOverrides!.find(
+        const customConfig = languageConfig[language].repoOverrides?.find(
           x => x.repo === repo
         );
         if (customConfig) {
+          console.log(`Discovered override config for ${repo}`);
```

Comment:
could this use @azizsonawalla's new logger that's being injected?

Reply:
oh wow good catch. This won't be the last time I make that mistake.